### PR TITLE
New aggregators: `delta` and `stdev`

### PR DESCRIPTION
**Pull request by guillaumeautran**
Implements two new aggregators:
- `delta`: reports the difference between min and max using the following formula: `delta := max - min`
- `stdev`: computes the standard deviation as the square root of the average of the squared deviations of the values from their average value.

Issue: #189

### CODE REVIEW

**Comment by guillaumeautran** [comment]
Ping for review: @Dieterbe 
Thanks.

File: aggregator/aggregator.go [hunk 1]
```diff
@@ -3,28 +3,36 @@ package aggregator
...
 		panic("avg() called in aggregator with 0 terms")
 	}
 	return Sum(in) / float64(len(in))
 }
 
+func Delta(in []float64) float64 {
+	if len(in) == 0 {
+		panic("delta() called in aggregator with 0 terms")
+	}
+	return Max(in) - Min(in)
```
Comment (Dieterbe) [inline aggregator/aggregator.go]: this works but can be done faster by not calling Min and Max, instead having 1 loop to iterate over the data once.

Comment (guillaumeautran) [inline aggregator/aggregator.go]: Yeah, I hesitated and took the shortest path first. 
Anyway, re-worked that part now to have a single loop. Thanks.

File: aggregator/aggregator.go [hunk 2]
```diff
@@ -3,28 +3,46 @@ package aggregator
...
+	max := in[0]
+	for _, val := range in {
+		if val > max {
+			max = val
+		} else if val < min {
+			min = val
+		}
+	}
+
+	return Max(in) - Min(in)
```
Comment (Dieterbe) [inline aggregator/aggregator.go]: i think you forgot to change this line.

Comment (guillaumeautran) [inline aggregator/aggregator.go]: D'oh! 
Thanks!

**Comment by Dieterbe** [comment]
great work @guillaumeautran this looks great ! thanks for the test also :+1:

**Comment by guillaumeautran** [comment]
Hey @Dieterbe , when are you planning on doing your next release for this package?

**Comment by Dieterbe** [comment]
@guillaumeautran for you, I made https://github.com/graphite-ng/carbon-relay-ng/releases/tag/0.8.8 :)

**Comment by guillaumeautran** [comment]
Thanks! I feel special now :D

**Comment by Dieterbe** [comment]
i just had to run a new CI job to get the new packages pushed with proper version. please let me know how the 0.8.8 deployment goes :)

**Pull request merged**